src/main/common/log.c

```diff
@@ -61,6 +61,11 @@ PG_RESET_TEMPLATE(logConfig_t, logConfig,
     .topics = SETTING_LOG_TOPICS_DEFAULT
 );
 
+#if defined(USE_BOOTLOG)
+char dmesg_buffer[USE_BOOTLOG];
+char *dmesg_head = dmesg_buffer;
```

Comment:
nit-pick: as much as i appreciate the linux reference, i would like to see dmesg replaced with bootlog everywhere for consistency.